http/CHANGELOG.md

```diff
@@ -2,6 +2,54 @@
 
 Changelog for `twilight-http`.
 
+## [0.4.3] - 2021-06-12
+
+### Additions
+
+Added new `ErrorCode`s:
+
+- `20031 UnallowedWordsForPublicStage` ([#886] - [@BlackHoleFox])
+- `30018 MaximumAnimatedEmojisReached` ([#918] - [@7596ff])
+- `30019 MaximumGuildMembersReached` ([#918] - [@7596ff])
+
+Added new request methods ([#867] - [@7596ff]):
+
+- `CreateStageInstance::privacy_level`
+- `UpdateStageInstance::privacy_level`
+- `UpdateStageInstance::topic`
+
+Note: it is currently impossible to change only the `privacy_level` of an existing stage instance.
+The change required to fix this is breaking, and will be fixed in a later version.
+
+### Changes
+
+The following dependencies have been removed:
+- `bytes` ([#909] - [@vivian])
+- `futures-util` ([#912] - [@vivian])
+- `native-tls` ([#911] - [@vivian])
+- `serde_repr` ([#920] - [@vivian])
+
+`Client::update_guild_channel_positions` was changed ([#880] - [@7596ff]):
+- It now accepts an `Iterator<Item = impl Into<Position>>`.
+- `twilight_http::request::guild::update_guild_channel_positions::Position` is now `pub`, and has new
+fields `lock_permissions` and `parent_id`.
+- To prevent a breaking change, a `From<(ChannelId, u64)> for Position` impl has been added.
```

Comment:
Same for this, it probably shouldn't be mentioned if we say the parameter is now more generic